railway fail

@claude what do I need to do to get this website working? I don't have a coding background so explain the steps simply. let me know what I need to do. the railway build failed so fix the all the problems. here are the logs: 

2026-04-24T05:03:07.413813402Z [inf]  
[notice] A new release of pip is available: 24.0 -> 26.0.1
[notice] To update, run: /app/venv/bin/python3 -m pip install --upgrade pip

2026-04-24T05:03:09.328366435Z [inf]  Done!

2026-04-24T05:03:10.240650822Z [inf]  sh build.sh
2026-04-24T05:03:17.862435404Z [inf]  copy /app
2026-04-24T05:03:22.085113451Z [inf]  copy /app
2026-04-24T05:03:22.087646182Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-24T05:03:22.283396163Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-24T05:03:22.285899485Z [inf]  exporting to docker image format
2026-04-24T05:03:22.285954073Z [inf]  exporting to image
2026-04-24T05:03:35.455583805Z [inf]  importing to docker
2026-04-24T05:03:36.438428914Z [inf]  [auth] sharing credentials for production-us-west2.railway-registry.com
2026-04-24T05:03:36.438487389Z [inf]  [auth] sharing credentials for production-us-west2.railway-registry.com
2026-04-24T05:03:49.015594470Z [inf]  importing to docker
2026-04-24T05:04:18.646306472Z [inf]  [92mBuild time: 131.87 seconds[0m
2026-04-24T05:04:28.630824489Z [inf]  
2026-04-24T05:04:28.630826897Z [inf]  [35m====================
Starting Healthcheck
====================
[0m
2026-04-24T05:04:28.630828052Z [inf]  [37mPath: /api/health[0m
2026-04-24T05:04:28.630828611Z [inf]  [37mRetry window: 5m0s[0m
2026-04-24T05:04:28.630829370Z [inf]  
2026-04-24T05:04:30.848949044Z [inf]  [93mAttempt #1 failed with service unavailable. Continuing to retry for 4m57s[0m
2026-04-24T05:04:34.160749356Z [inf]  [93mAttempt #2 failed with service unavailable. Continuing to retry for 4m54s[0m
2026-04-24T05:04:37.346442635Z [inf]  [93mAttempt #3 failed with service unavailable. Continuing to retry for 4m51s[0m
2026-04-24T05:04:43.612449509Z [inf]  [93mAttempt #4 failed with service unavailable. Continuing to retry for 4m45s[0m
2026-04-24T05:04:52.949212782Z [inf]  [93mAttempt #5 failed with service unavailable. Continuing to retry for 4m35s[0m
2026-04-24T05:05:18.987003738Z [inf]  [93mAttempt #6 failed with service unavailable. Continuing to retry for 4m9s[0m
2026-04-24T05:05:59.730257794Z [inf]  [93mAttempt #7 failed with service unavailable. Continuing to retry for 3m28s[0m
2026-04-24T05:06:39.769105830Z [inf]  [93mAttempt #8 failed with service unavailable. Continuing to retry for 2m48s[0m
2026-04-24T05:07:19.809360401Z [inf]  [93mAttempt #9 failed with service unavailable. Continuing to retry for 2m8s[0m
2026-04-24T05:08:00.100069870Z [inf]  [93mAttempt #10 failed with service unavailable. Continuing to retry for 1m28s[0m
2026-04-24T05:08:40.142679782Z [inf]  [93mAttempt #11 failed with service unavailable. Continuing to retry for 48s[0m
2026-04-24T05:09:20.182630701Z [inf]  [93mAttempt #12 failed with service unavailable. Continuing to retry for 8s[0m
2026-04-24T05:09:20.327269351Z [inf]  
2026-04-24T05:09:20.327273113Z [inf]  [91m1/1 replicas never became healthy![0m
2026-04-24T05:09:20.327273669Z [inf]  [91mHealthcheck failed![0m



deploy logs: 
2026-04-24T05:05:02.466384063Z [err]  syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.466386217Z [err]      from auth import bp as auth_bp
2026-04-24T05:05:02.466388867Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.466391988Z [err]    File "/app/backend/auth.py", line 163, in <module>
2026-04-24T05:05:02.466395384Z [err]                                               ^
2026-04-24T05:05:02.466397633Z [err]      _ensure_password_reset_tokens_table()
2026-04-24T05:05:02.466400151Z [err]  
2026-04-24T05:05:02.466402465Z [err]    File "/app/backend/auth.py", line 150, in _ensure_password_reset_tokens_table
2026-04-24T05:05:02.466404704Z [err]  [2026-04-24 05:05:02 +0000] [5] [INFO] Worker exiting (pid: 5)
2026-04-24T05:05:02.466407849Z [err]      db.execute(
2026-04-24T05:05:02.466409857Z [err]  [2026-04-24 05:05:02 +0000] [6] [ERROR] Exception in worker process
2026-04-24T05:05:02.466412520Z [err]    File "/app/backend/db.py", line 73, in execute
2026-04-24T05:05:02.466414900Z [err]  Traceback (most recent call last):
2026-04-24T05:05:02.466417448Z [err]      wrapper.execute(sql, params)
2026-04-24T05:05:02.466420437Z [err]    File "/app/backend/db.py", line 25, in execute
2026-04-24T05:05:02.466423295Z [err]      self._cursor.execute(sql, params)
2026-04-24T05:05:02.466426338Z [err]    File "/app/venv/lib/python3.11/site-packages/psycopg2/extras.py", line 236, in execute
2026-04-24T05:05:02.466429059Z [err]      return super().execute(query, vars)
2026-04-24T05:05:02.466433339Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.466436159Z [err]  psycopg2.errors.SyntaxError: syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.466438974Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.466441960Z [err]                                               ^
2026-04-24T05:05:02.467675013Z [err]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.467677708Z [err]      self.callable = self.load()
2026-04-24T05:05:02.467679765Z [err]    File "/mise/installs/python/3.11.15/lib/python3.11/importlib/__init__.py", line 126, in import_module
2026-04-24T05:05:02.467684249Z [err]                      ^^^^^^^^^^^
2026-04-24T05:05:02.467685265Z [err]      return _bootstrap._gcd_import(name[level:], package, level)
2026-04-24T05:05:02.467689017Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
2026-04-24T05:05:02.467693001Z [err]      return self.load_wsgiapp()
2026-04-24T05:05:02.467693947Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 713, in spawn_worker
2026-04-24T05:05:02.467696585Z [err]             ^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.467699982Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
2026-04-24T05:05:02.467704001Z [err]      return util.import_app(self.app_uri)
2026-04-24T05:05:02.467706388Z [err]      worker.init_process()
2026-04-24T05:05:02.467708471Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.467713389Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/util.py", line 377, in import_app
2026-04-24T05:05:02.467714800Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 136, in init_process
2026-04-24T05:05:02.467717746Z [err]      mod = importlib.import_module(module)
2026-04-24T05:05:02.467721175Z [err]      self.load_wsgi()
2026-04-24T05:05:02.467725700Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2026-04-24T05:05:02.467730582Z [err]      self.wsgi = self.app.wsgi()
2026-04-24T05:05:02.467734918Z [err]                  ^^^^^^^^^^^^^^^
2026-04-24T05:05:02.467739126Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 66, in wsgi
2026-04-24T05:05:02.469706946Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.469710537Z [err]    File "/app/backend/db.py", line 73, in execute
2026-04-24T05:05:02.469715902Z [err]    File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2026-04-24T05:05:02.469719305Z [err]      wrapper.execute(sql, params)
2026-04-24T05:05:02.469722475Z [err]    File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2026-04-24T05:05:02.469724940Z [err]    File "/app/backend/db.py", line 25, in execute
2026-04-24T05:05:02.469729374Z [err]    File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2026-04-24T05:05:02.469731503Z [err]      self._cursor.execute(sql, params)
2026-04-24T05:05:02.469736310Z [err]    File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2026-04-24T05:05:02.469737133Z [err]    File "/app/venv/lib/python3.11/site-packages/psycopg2/extras.py", line 236, in execute
2026-04-24T05:05:02.469743258Z [err]      return super().execute(query, vars)
2026-04-24T05:05:02.469745936Z [err]    File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2026-04-24T05:05:02.469748762Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.469753535Z [err]    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2026-04-24T05:05:02.469755429Z [err]  psycopg2.errors.SyntaxError: syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.469759095Z [err]    File "/app/backend/app.py", line 10, in <module>
2026-04-24T05:05:02.469763148Z [err]      from auth import bp as auth_bp
2026-04-24T05:05:02.469766987Z [err]    File "/app/backend/auth.py", line 163, in <module>
2026-04-24T05:05:02.469770421Z [err]      _ensure_password_reset_tokens_table()
2026-04-24T05:05:02.469773929Z [err]    File "/app/backend/auth.py", line 150, in _ensure_password_reset_tokens_table
2026-04-24T05:05:02.469777155Z [err]      db.execute(
2026-04-24T05:05:02.471457195Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.471464241Z [err]                                               ^
2026-04-24T05:05:02.471469845Z [err]  
2026-04-24T05:05:02.471474724Z [err]  syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.471480484Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.471485297Z [err]                                               ^
2026-04-24T05:05:02.471490342Z [err]  
2026-04-24T05:05:02.471498328Z [err]  [2026-04-24 05:05:02 +0000] [6] [INFO] Worker exiting (pid: 6)
2026-04-24T05:05:02.517308316Z [err]  [2026-04-24 05:05:02 +0000] [1] [ERROR] Worker (pid:4) exited with code 3.
2026-04-24T05:05:02.566136225Z [err]                      ^^^^^^^^^^^
2026-04-24T05:05:02.566143751Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
2026-04-24T05:05:02.566147756Z [err]      return self.load_wsgiapp()
2026-04-24T05:05:02.566150974Z [err]             ^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.566154686Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
2026-04-24T05:05:02.566159247Z [err]      return util.import_app(self.app_uri)
2026-04-24T05:05:02.566163220Z [err]  [2026-04-24 05:05:02 +0000] [7] [ERROR] Exception in worker process
2026-04-24T05:05:02.566168744Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.566171996Z [err]  Traceback (most recent call last):
2026-04-24T05:05:02.566174904Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/util.py", line 377, in import_app
2026-04-24T05:05:02.566180092Z [err]      mod = importlib.import_module(module)
2026-04-24T05:05:02.566180475Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 713, in spawn_worker
2026-04-24T05:05:02.566185011Z [err]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.566187609Z [err]      worker.init_process()
2026-04-24T05:05:02.566191738Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 136, in init_process
2026-04-24T05:05:02.566195689Z [err]      self.load_wsgi()
2026-04-24T05:05:02.566200366Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2026-04-24T05:05:02.566204041Z [err]      self.wsgi = self.app.wsgi()
2026-04-24T05:05:02.566209187Z [err]                  ^^^^^^^^^^^^^^^
2026-04-24T05:05:02.566213493Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 66, in wsgi
2026-04-24T05:05:02.566217866Z [err]      self.callable = self.load()
2026-04-24T05:05:02.574398177Z [err]    File "/mise/installs/python/3.11.15/lib/python3.11/importlib/__init__.py", line 126, in import_module
2026-04-24T05:05:02.574402765Z [err]      return _bootstrap._gcd_import(name[level:], package, level)
2026-04-24T05:05:02.574405910Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.574409023Z [err]    File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2026-04-24T05:05:02.574411743Z [err]    File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2026-04-24T05:05:02.574414658Z [err]    File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2026-04-24T05:05:02.574417397Z [err]    File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2026-04-24T05:05:02.574420139Z [err]    File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2026-04-24T05:05:02.574422740Z [err]    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2026-04-24T05:05:02.574425357Z [err]    File "/app/backend/app.py", line 10, in <module>
2026-04-24T05:05:02.574427946Z [err]      from auth import bp as auth_bp
2026-04-24T05:05:02.574431840Z [err]    File "/app/backend/auth.py", line 163, in <module>
2026-04-24T05:05:02.574434472Z [err]      _ensure_password_reset_tokens_table()
2026-04-24T05:05:02.574437081Z [err]    File "/app/backend/auth.py", line 150, in _ensure_password_reset_tokens_table
2026-04-24T05:05:02.574439849Z [err]      db.execute(
2026-04-24T05:05:02.574442455Z [err]    File "/app/backend/db.py", line 73, in execute
2026-04-24T05:05:02.574445037Z [err]      wrapper.execute(sql, params)
2026-04-24T05:05:02.574447706Z [err]    File "/app/backend/db.py", line 25, in execute
2026-04-24T05:05:02.574450418Z [err]      self._cursor.execute(sql, params)
2026-04-24T05:05:02.574453117Z [err]    File "/app/venv/lib/python3.11/site-packages/psycopg2/extras.py", line 236, in execute
2026-04-24T05:05:02.576594915Z [err]      return super().execute(query, vars)
2026-04-24T05:05:02.576598893Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T05:05:02.576602475Z [err]  psycopg2.errors.SyntaxError: syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.576606185Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.576610204Z [err]                                               ^
2026-04-24T05:05:02.576614219Z [err]  
2026-04-24T05:05:02.576618155Z [err]  syntax error at or near "AUTOINCREMENT"
2026-04-24T05:05:02.576642326Z [err]  LINE 3:               id INTEGER PRIMARY KEY AUTOINCREMENT,
2026-04-24T05:05:02.576646270Z [err]                                               ^
2026-04-24T05:05:02.576649616Z [err]  
2026-04-24T05:05:02.576653308Z [err]  [2026-04-24 05:05:02 +0000] [7] [INFO] Worker exiting (pid: 7)
2026-04-24T05:05:02.636960772Z [err]  [2026-04-24 05:05:02 +0000] [1] [INFO] Worker (pid:5) was sent SIGTERM!
2026-04-24T05:05:02.636964144Z [err]  [2026-04-24 05:05:02 +0000] [1] [INFO] Worker (pid:6) was sent SIGTERM!
2026-04-24T05:05:02.853886086Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 271, in handle_chld
2026-04-24T05:05:02.853894823Z [err]      self.reap_workers()
2026-04-24T05:05:02.853899760Z [err]  [2026-04-24 05:05:02 +0000] [1] [ERROR] Worker (pid:7) exited with code 3.
2026-04-24T05:05:02.853902518Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 641, in reap_workers
2026-04-24T05:05:02.853905804Z [err]  Traceback (most recent call last):
2026-04-24T05:05:02.853909373Z [err]      raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2026-04-24T05:05:02.853912039Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 246, in run
2026-04-24T05:05:02.853916961Z [err]  gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
2026-04-24T05:05:02.853919040Z [err]      handler()
2026-04-24T05:05:02.853923522Z [err]  
2026-04-24T05:05:02.853927001Z [err]  During handling of the above exception, another exception occurred:
2026-04-24T05:05:02.853930876Z [err]  
2026-04-24T05:05:02.853934212Z [err]  Traceback (most recent call last):
2026-04-24T05:05:02.853937842Z [err]    File "/app/venv/bin/gunicorn", line 8, in <module>
2026-04-24T05:05:02.853942201Z [err]      sys.exit(run())
2026-04-24T05:05:02.853946604Z [err]               ^^^^^
2026-04-24T05:05:02.853949733Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 66, in run
2026-04-24T05:05:02.853953368Z [err]      WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]", prog=prog).run()
2026-04-24T05:05:02.853960024Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 235, in run
2026-04-24T05:05:02.853963671Z [err]      super().run()
2026-04-24T05:05:02.853967026Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 71, in run
2026-04-24T05:05:02.853970428Z [err]      Arbiter(self).run()
2026-04-24T05:05:02.855225415Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 254, in run
2026-04-24T05:05:02.855230078Z [err]      self.halt(reason=inst.reason, exit_status=inst.exit_status)
2026-04-24T05:05:02.855233149Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 378, in halt
2026-04-24T05:05:02.855236307Z [err]      self.stop()
2026-04-24T05:05:02.855239590Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 458, in stop
2026-04-24T05:05:02.855242789Z [err]      self.reap_workers()
2026-04-24T05:05:02.855245863Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 641, in reap_workers
2026-04-24T05:05:02.855251705Z [err]      raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2026-04-24T05:05:02.855255196Z [err]  gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>


render logs:
2026-04-24T04:50:34.215691476Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2026-04-24T04:50:40.827592218Z ==> Running 'cd backend && python app.py'
2026-04-24T04:50:41.023213723Z Traceback (most recent call last):
2026-04-24T04:50:41.023240055Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T04:50:41.023251455Z     import requests
2026-04-24T04:50:41.023258296Z ModuleNotFoundError: No module named 'requests'
2026-04-24T05:01:52.195321505Z ==> Deploying...
2026-04-24T05:01:52.2726976Z ==> Setting WEB_CONCURRENCY=1 by default, based on available CPUs in the instance
2026-04-24T05:02:21.516643835Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T05:02:21.516656885Z     import requests
2026-04-24T05:02:21.516660805Z ModuleNotFoundError: No module named 'requests'
2026-04-24T05:02:22.101791601Z ==> Exited with status 1
2026-04-24T05:02:22.104145158Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2026-04-24T05:02:28.632243999Z ==> Running 'cd backend && python app.py'
2026-04-24T05:02:28.867709362Z Traceback (most recent call last):
2026-04-24T05:02:28.867723813Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T05:02:28.867816105Z     import requests
2026-04-24T05:02:28.867820435Z ModuleNotFoundError: No module named 'requests'
2026-04-24T05:02:31.327383493Z ==> Running 'cd backend && python app.py'
2026-04-24T05:02:52.105460169Z ==> Running 'cd backend && python app.py'
2026-04-24T05:02:52.398913276Z Traceback (most recent call last):
2026-04-24T05:02:52.398948067Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T05:02:52.398997898Z     import requests
2026-04-24T05:02:52.399002568Z ModuleNotFoundError: No module named 'requests'
2026-04-24T05:02:56.93629519Z ==> No open ports detected, continuing to scan...
2026-04-24T05:02:56.998555706Z ==> Docs on specifying a port: https://render.com/docs/web-services#port-binding
2026-04-24T05:02:57.066698866Z  * Serving Flask app 'app'
2026-04-24T05:02:57.066723236Z  * Debug mode: on
2026-04-24T05:02:57.3255162Z WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
2026-04-24T05:02:57.32553423Z  * Running on all addresses (0.0.0.0)
2026-04-24T05:02:57.32553788Z  * Running on http://127.0.0.1:10000
2026-04-24T05:02:57.32554106Z  * Running on http://10.199.223.99:10000
2026-04-24T05:02:57.32554484Z Press CTRL+C to quit
2026-04-24T05:02:57.330660779Z  * Restarting with stat
2026-04-24T05:03:07.527029863Z  * Debugger is active!
2026-04-24T05:03:07.527733088Z  * Debugger PIN: 155-394-848
2026-04-24T05:03:55.967734428Z ==> Deploying...
2026-04-24T05:03:56.017380034Z ==> Setting WEB_CONCURRENCY=1 by default, based on available CPUs in the instance
2026-04-24T05:04:24.080950889Z ==> Running 'cd backend && python app.py'
2026-04-24T05:04:24.291824551Z Traceback (most recent call last):
2026-04-24T05:04:24.291838921Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T05:04:24.291858752Z     import requests
2026-04-24T05:04:24.291866182Z ModuleNotFoundError: No module named 'requests'
2026-04-24T05:04:35.167276594Z ==> Exited with status 1
2026-04-24T05:04:35.175152436Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2026-04-24T05:04:41.774152693Z ==> Running 'cd backend && python app.py'
2026-04-24T05:04:42.059456742Z Traceback (most recent call last):
2026-04-24T05:04:42.059481053Z   File "/opt/render/project/src/backend/app.py", line 2, in <module>
2026-04-24T05:04:42.059641718Z     import requests
2026-04-24T05:04:42.059669499Z ModuleNotFoundError: No module named 'requests'

--- a/backend/auth.py
+++ b/backend/auth.py
@@ -150,7 +150,7 @@ def _ensure_password_reset_tokens_table():
         db.execute(
             """
             CREATE TABLE IF NOT EXISTS password_reset_tokens (
-              id INTEGER PRIMARY KEY AUTOINCREMENT,
+              id SERIAL PRIMARY KEY,
               user_id INTEGER NOT NULL,
               token TEXT UNIQUE NOT NULL,
               expires_at INTEGER NOT NULL,

